Makefile

```diff
@@ -143,12 +143,10 @@ test-npm: $(NODE_EXE)
 	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
 	     npm_config_prefix="$(shell pwd)/npm-prefix" \
 	     npm_config_tmp="$(shell pwd)/npm-tmp" \
-	     ../../$(NODE_EXE) cli.js install --ignore-scripts
-	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
-	     npm_config_prefix="$(shell pwd)/npm-prefix" \
-	     npm_config_tmp="$(shell pwd)/npm-tmp" \
+	     ../../$(NODE_EXE) cli.js install --ignore-scripts && \
 	     ../../$(NODE_EXE) cli.js run-script test-all && \
 	     ../../$(NODE_EXE) cli.js prune --prod && \
+	     rm -rf node_modules/.bin && \
 	     cd ../.. && \
 	     rm -rf npm-cache npm-tmp npm-prefix
 
```

Comment:
Could you also fix the indentation of this scope? Looks off by a space. Also, tabs is common in Makefiles.